interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: A conditional statement controls the execution of code based on a condition. If a condition is true then one block of code will run. If the condition is false then an alternate code block can run.
 
-Researched answer:
+Researched answer: A conditional statement is code that instructs the computer is perform a task provided that certain conditions are met. These statements allow programs to simulate human behavior by taking different actions based on the decision made.
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a program developed by Linus Torvalds to solve a huge problem in software development known as version control. GitHub is a 'hub' for code. Using the git version control software, developers can upload their code to GitHub. Many people can make changes and add features to the code and git handles how each contribution is made.
 
-Researched answer:
+Researched answer: Git allows you to keep track and manage your programs history. This is helpful because if a bug is introduced onto the code, you can revert those changes and go about a different way to find a solution. GitHub allows you to host and manage your git repositories. This makes it easy for many people to contribute to the same project.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: !, &&, ||
 
-Researched answer:
+Researched answer: && (AND) returns true if and only if both values on either side of the operator are true. || (OR) will return true if one of two values are true. The bang operator (! NOT) will invert a value.
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: An index represents an items position in a collection. An index points to a given value.
 
-Researched answer:
+Researched answer: An index is typically a number and is used as an address in an ordered collection. A value can be stored in an ordered collection at a given index.
```

Comment:
This would be a great place to broaden your current knowledge. Try to find something you don't know about the question.